examples/run_composer_trainer_tpu.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+import sys
+import tempfile
+import warnings
+from typing import Type
+
+from composer.loggers.logger import LogLevel
+from composer.trainer.trainer_hparams import TrainerHparams
+from composer.utils import dist
+import torch
+
+def train():
+
+    if len(sys.argv) == 1:
```

Comment:
Curious why do we need a separate entrypoint for TPUs? I think with the composer launcher script we shouldn't need one?

Reply:
I added some details in the design doc, if you have any other questions we can discuss offline.